### DISCUSSION TITLE
Anyway to set custom delimiters?

### DISCUSSION THREAD

**Discussion by ShahrukhAhmed89** [vuejs/petite-vue#32] [Q&A]
For templating engines like liquid that use handlebar syntax, is there a way to set custom delimiters?

**Comment by sqllyw** [comment]
if handledbar, then escape as follow works for me:
````
<div v-scope="{ count: 0 }">
     \{{count}}
      <button @click="count++">inc</button>
</div>

````

**Reply by ShahrukhAhmed89** [reply]
I can't do that since liquid renders handlebars server-side. So the resulting output is just slash.

**Reply by niutech** [reply]
@ShahrukhAhmed89 In Liquid you can put `{{count}}` between `{% raw %}` and `{% endraw %}`.

**Reply by ShahrukhAhmed89** [reply]
Thanks. Will try this.
